pkg/inventory/inventory-client_test.go

```diff
@@ -291,11 +291,11 @@ func TestReplace(t *testing.T) {
 	// Client and server dry-run do not throw errors.
 	invClient, err := NewInventoryClient(tf, WrapInventoryObj, InvInfoToConfigMap)
 	require.NoError(t, err)
-	err = invClient.Replace(copyInventory(), object.ObjMetadataSet{}, common.DryRunClient)
+	err = invClient.Replace(copyInventory(), object.ObjMetadataSet{}, nil, common.DryRunClient)
```

Comment:
These tests are just ignoring the new input. Please add new tests that exercise the new status field.

Reply:
done